test/cp.js

```diff
@@ -756,3 +756,29 @@ test('should not overwrite recently created files (not give error no-force mode)
   // Ensure First file is copied
   t.is(shell.cat(`${t.context.tmp}/file1`).toString(), 'test1');
 });
+
+test('cp -p should preserve mode, ownership, and timestamp', t => {
+  const result = shell.cp('-p', 'test/resources/file1', `${t.context.tmp}/preservedFile1`);
```

Comment:
I'll follow-up to this to `touch()` the original file. Otherwise, we might have a race where the test setup creates `file1` and the test nearly-immediately copies it--depending on the timing/precision, a regular `cp()` call might even pass this test.

We should probably also chmod the file too, to double-check the mode gets copied correctly (and it isn't the default mode).